### ISSUE TITLE
lerna lite get `packages` from `workspaces` in `package.json`

### ISSUE THREAD

**Issue by Naymi** [lerna-lite/lerna-lite#645]
### Describe the bug

lerna 7 by default use workspaces from `package.json`
lerna-lite by default use [`packages/*`](https://github.com/lerna-lite/lerna-lite/wiki/lerna.json#:~:text=By%20default%2C%20the,packages%20as%20well.)

### Reproduction

[-https://github.com/lerna/lerna/releases#:~:text=Workspaces%20are%20used%20by%20default%20and%20useWorkspaces%20has%20been%20removed](https://github.com/lerna/lerna/releases#:~:text=Workspaces%20are%20used%20by%20default%20and%20useWorkspaces%20has%20been%20removed)

### Lerna config and logs

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

### Environment Info

 System:
    OS: Linux 5.19 Ubuntu 22.10 22.10 (Kinetic Kudu)
    CPU: (24) x64 12th Gen Intel(R) Core(TM) i9-12900
    Memory: 25.74 GB / 62.62 GB
    Container: Yes
    Shell: 5.2.2 - /bin/bash
  Binaries:
    Node: 18.16.0 - /usr/bin/node
    npm: 9.6.7 - /usr/bin/npm
  Browsers:
    Chrome: 114.0.5735.106
    Firefox: 113.0.2

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/lerna-lite/lerna-lite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/lerna-lite/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/lerna-lite/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/lerna-lite/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
Lerna-Lite was created over a year ago when Lerna was at v4 and I have never made the change that the new Lerna team did with `useWorkspaces` because I disagree with it. That flag was originally created **only** to support Yarn workspaces and it was meant to read the `workspaces` field in `package.json` and nothing else. However some users did not want to include all content of that said `workspaces`, a good example is Jest (which is the biggest project using Lerna-Lite), Jest is using Yarn v3 (aka Berry) and their `worspaces` in `package.json` is the following

(link to live code [here](https://github.com/jestjs/jest/blob/6460335f88cee3dcb9d29c49d55ab02b9d83f994/package.json#L122-L126))
```js
"workspaces": [
    "packages/*",
    "website",
    "examples/*"
],
```
but Jest is only every interested into `package/*` since `website` and `examples/*` are not at all related to the project versioning of releases. So their `lerna.json` config to only release what is included in `packages` is to simply use `packages: ['package/*']` and website and examples are automatically skipped since they're not included in what Lerna-Lite reads for releasing new versions, so this works correctly and they're happy with it.

Now going back to Lerna v6-7 which automatically reads the workspaces, what they're doing is automatic yes but it forces the use of possibly unrelated folders (like Jest website & examples) and I'm not exactly sure how users will be able to exclude folders to be read with their approach (maybe with gitignore? I don't know). If for example Jest would switch to Lerna v7, that would be a breaking change for their project because again website and examples shouldn't be read (even though they are included in their yarn workspaces). I try to sync PRs that I can from Lerna into Lerna-Lite, but I disagree with some, like this one. It's also becoming harder to keep in sync with them since their code structure changed entirely and it's becoming more and more another Nx product since a lot of the new codebase uses Nx now.

TLDR
in short, I disagree with the new implementation of `useWorkspaces` by the new Lerna team and that is why I never implemented this change in Lerna-Lite. 

I'm open to discussion but as it stands now, I'm not really interested in doing this change/request since that would be a breaking change and would require a lot more change than you expect.

**Comment by Naymi** [comment]
thanks for the detailed answer!
